examples/json-csv/json_csv.bal

```diff
@@ -0,0 +1,72 @@
+import ballerina/io;
+import ballerina/log;
+
+// Converts a json into an array of strings
+// Returns the result as a tuple which contains the headers and fields
+function getFields(json rec) returns (string[], string[]) {
+    int count =0;
+    string [] headers = [];
+    string [] fields = [];
+    headers = rec.getKeys();
+    foreach var field in headers {
+        fields[count] = rec[field].toString();
+        count = count + 1;
+    }
+    return (headers, fields);
+}
+
+// Example function which writes json content to csv
+function writeCsv(json content, string path) returns error? {
+    io:WritableCSVChannel csvch = io:openWritableCsvFile(path);
+    int recIndex =0;
+    int recLen = content.length();
+    while (recIndex < recLen) {
+        (string [], string []) result = getFields(content[recIndex]);
+        var (headers, fields) = result;
+        if (recIndex == 0) {
+             var wResult1 = check csvch.write(headers);
```

Comment:
Shall we change the variable names since it is an example. I don't see where these variables are used later.

Reply:
write contains error|() we need to assign the result to a variable ( which is enforced at compile time) if there's an error this will be propagated with the check

Reply:
If it is not used you can ignore using `_`